dataland-frontend/src/components/forms/CreateP2pDataset.vue

```diff
@@ -243,6 +267,10 @@ export default defineComponent({
     async postP2pData(): Promise<void> {
       this.messageCounter++;
       try {
+        this.filterFieldSpecificDocumentsByEnabledFieldOwnership();
```

Comment:
Why do we need this?

Reply:
P2P has these sectors which can be enabled and disabled. If I e.g. enable LivestockFarming and put a document into a base datapoint in the LivestockFarming sector that document should be uploaded. But if I disable the sector again it should no longer be uploaded because the corresponding base data point does no longer exist. These fields are filtered out in this function.

Reply:
ok